src/app/areas.json

```diff
@@ -1,5 +1,5 @@
 {
     "areas": [
-        [ "Deutschland", "Berlin" ]
+        [ "This is not used anymore, because python had problems to correctly load letters like Ö from the json", "Instead, change cities at get_data.py in main()"]
```

Comment:
If it isnt used delete it (maybe dennis has an idea to pass the json in the correct encoding though -> Using the json makes the service more easily extendable so it would be nicer if the json can be used)

Reply:
I will check it. There might be a quick solution, but if there isn't I will just delete the json file/code.